bot/exts/info/help.py

```diff
@@ -264,7 +264,8 @@ async def subcommand_not_found(self, command: Command, string: str) -> "HelpQuer
 
     async def send_error_message(self, error: HelpQueryNotFound) -> None:
         """Send the error message to the channel."""
-        embed = Embed(colour=Colour.red(), title=str(error))
+        # 2257 the title can be too big. so we trim it
```

Comment:
```suggestion
        # Embed titles have a length limits, so we trim here.
```
having a number out of context in a comment isn't very useful.

I know it links to the issue you are fixing, but in 2 months when someone looks at this code, they won't.
Linking the comment to the issue doesn't really add anything, since we already explain why we need to do this.